package.json

```diff
@@ -125,6 +125,7 @@
     "passport-oauth2": "^1.6.0",
     "redis": "^3.1.1",
     "superagent": "^6.1.0",
+    "tar": "^4.4.15",
```

Comment:
We don't want to keep `tar` in our project's dependencies long-term because (amongst other reasons) it'll eventually affect our ability to update to the latest version of the Pact libraries once they become available (`pact-js-core` version's `main` has already updated to `tar` 6.1.4). Can we create a tech debt ticket to track this?

I'd also say that it's worth adding a `"//": "tar is not a direct dependency of our project; we’ve only added it here to remove a security vulnerability introduced by a transitive dependency. This should be removed; see IC-xxxx"` comment too.

Reply:
I agree and great suggestion. I've created a JIRA ticket: https://dsdmoj.atlassian.net/browse/IC-2177
and I've added a comment.